src/ops/fuelup_toolchain/export.rs

```diff
@@ -0,0 +1,98 @@
+use crate::{
+    commands::toolchain::ExportCommand,
+    toolchain::Toolchain,
+    constants::FUEL_TOOLCHAIN_TOML_FILE,
+    toolchain_override::{self, OverrideCfg, ToolchainCfg, ToolchainOverride},
+};
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+use semver::Version;
+use anyhow::{bail, Result};
+use component::{self, Components};
+use tracing::info;
+
+pub fn export(command: ExportCommand) -> Result<()> {
+    let ExportCommand { name, force} = command;
+    let path = PathBuf::from("./").join(FUEL_TOOLCHAIN_TOML_FILE);
+    if !force && path.exists() {
+        bail!("{FUEL_TOOLCHAIN_TOML_FILE} already exists");
```

Comment:
Let's do a similar behavior to what's being done in https://github.com/FuelLabs/forc-wallet/pull/148

That is, if `--force` is not supplied, ask the user if they wish to override it. If it is, give the user a warning that the file is being overwritten.